sdk/keyvault/azure-keyvault-administration/samples/settings_operations.py

```diff
@@ -45,6 +45,7 @@
     if setting.setting_type == KeyVaultSettingType.BOOLEAN:
         boolean_setting = setting
     print(f"{setting.name}: {setting.value} (type: {setting.setting_type})")
+assert isinstance(boolean_setting, KeyVaultSetting)
```

Comment:
just wondering, what was the error that needed this isinstance check?

Reply:
This was to disambiguate the type, since we initially set `boolean_setting` to `None`. In retrospect, we could probably simplify this to `assert boolean_setting`